### PR TITLE
CloudTrail Customization

### PR DESCRIPTION
**Pull request by jricks92**
Adds parameters for various CloudTrail options. Adds ability to log to the local account as well as the centralized account.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+#### Security/Logging Account
+1. Login to your Security/Logging AWS account
+2. Create a New Stack in the CloudFormation console.
+3. Upload the `logging-account-cloudtrail.yaml` file as the template.
+4. Customize the parameters for your use case.
+5. Deploy the Stack
+6. Note the outputs for the KMS key ARN and S3 bucket name
+
+#### Non-Logging Accounts
```
Comment (dougireton) [inline README.md]:
```suggestion
#### Resource Accounts
```

File: logging-account-cloudtrail.yaml [hunk 2]
```diff
@@ -235,24 +318,28 @@ Resources:
...
-      S3BucketName: !Ref 'CloudTrailBucketName'
-      KMSKeyId: !Ref 'CloudTrailEncryptionKey'
+      IsMultiRegionTrail: !Ref MultiRegion
+      EventSelectors: 
+        - ReadWriteType: !Ref ReadWriteType
+      S3BucketName: !Ref CloudTrailBucketName
+      KMSKeyId: !If [ ShouldEncryptLogs, !GetAtt CloudTrailEncryptionKey.Arn, !Ref "AWS::NoValue" ] 
       TrailName: !Ref CloudTrailLogName
 Outputs:
   CloudTrailEncryptionKeyArn:
```
Comment (dougireton) [inline logging-account-cloudtrail.yaml:340]: Use a `Condition: ShouldEncryptLogs` key here instead to either add the Output or not. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html

**Review by dougireton — COMMENTED** [review]
Nice work @jricks92. Just had a couple of minor suggestions.

File: logging-account-cloudtrail.yaml (hunk 2 above)
Comment (jricks92) [reply logging-account-cloudtrail.yaml:340]: Good catch! I forgot about Conditions for Outputs. It's been added now.

File: README.md (hunk 1 above)
Comment (jricks92) [reply README.md]: Fixed :)

**Pull request merged**